livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -274,6 +290,15 @@ def _to_cartesia_options(opts: _TTSOptions) -> dict[str, Any]:
     else:
         voice["mode"] = "embedding"
         voice["embedding"] = opts.voice
+        
+    voice_controls = {}
```

Comment:
doing that will fix the Check Types CI
```suggestion
    voice_controls: dict = {}
```